logstash-core/lib/logstash/pipeline.rb

```diff
@@ -176,8 +179,10 @@ def stopped?
   end
 
   def start_workers
-    @inflight_batches = {}
-
+    @filter_queue_client.set_events_metric(metric.namespace([:stats, :events]))
```

Comment:
yeah, I often struggle on choosing between a `set_something(...)` method versus `something = ...` - in addition to the return value, there is also the multiple parameters issue.  In general I must say that I tend to prefer the `set_*` naming as I find it is more immediately obvious we are calling a setter and not assigning a local variable, for instance.